CMakeLists.txt

```diff
@@ -87,11 +89,25 @@ else()
   endforeach()
 endif()
 
+option(K2_USE_GLOG "Flag that chooses logger from [GLOG, NON-GLOG (loguru)]" OFF)
```

Comment:
@songmeixu , I still don't get the idea here why we need to bring in `loguru`? Does it will slove the issues 1-4 with `glog` you metioned before? Can you explain in detail how it solves those issues as I can't  figure it out by myself?

Reply:
I think I have commented on this statement before.

- Could you explain when should we enable this option?
- If it is fine to enable this option, why not to always use glog in K2, so that we can avoid introducing another logging library?